packages/alfa-scraper/src/scraper.ts

```diff
@@ -36,6 +36,17 @@ export class Scraper {
     return new Scraper(await browser);
   }
 
+  public static async with<T>(
+    mapper: Mapper<Scraper, Promise<T>>,
+    browser?: Promise<puppeteer.Browser>
+  ): Promise<T> {
+    const scraper = await this.of(browser);
```

Comment:
Using `this` inside a static method triggered all my alarms… Then I realised that `of` is also static…
```suggestion
    const scraper = await Scraper.of(browser);
```
🤔 hmm… unless we do need to trickle down class hierarchy and call the correct `of`…

Reply:
While the class has a private constructor and therefore cannot be subclassed, we _could_ call `Scraper.of()` just fine. But, for the sake of consistency with classes that _can_ be subclassed, using `this` makes sure we're always calling the right class.